src/csharp/Grpc.Core.Api/Grpc.Core.Api.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageTags>gRPC RPC HTTP/2</PackageTags>
     <VersionPrefix>$(GrpcCsharpVersion)</VersionPrefix>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <LangVersion>8.0</LangVersion>
```

Comment:
It seems like this shouldn't be a strict requirement. We could implement the extension method without C# 8.0 if we wanted, right?  If that's the case, the only blocker seems to be that Microsoft.Bcl.AsyncInterfaces is not stable.

Reply:
It isn't required. Without it the ReadAllAsync method would need to be manually implemented instead of relying on the `yield return` and compiler.

Reply:
> If that's the case, the only blocker seems to be that Microsoft.Bcl.AsyncInterfaces is not stable.

Targeting netstandard2.1 avoids the need for `Microsoft.Bcl.AsyncInterfaces`. ReadAllAsync could be added to a netstandard2.1 target now and then included in netstandard2.0 post 3.0.